xml/html_title.xml

```diff
@@ -3770,4 +3770,24 @@
     <param pos="0" name="service.cpe23" value="cpe:/a:apache:solr:-"/>
   </fingerprint>
 
+  <fingerprint pattern="^pfSense - Login$">
+    <description>pfSense Firewall</description>
+    <example>pfSense - Login</example>
+    <param pos="0" name="service.vendor" value="pfSense"/>
+    <param pos="0" name="service.product" value="pfSense"/>
```

Comment:
I believe that the actual product here, per the `Server` header is `nginx`. I'm checking our data to be sure. I should be able to provide an update in an hour or so.

Reply:
Here are the frequency counts values for the `Server` header for Sonar responses where the title is `pfSense - Login`. These were taken from an HTTP study vs `80/tcp` on Jan 23, 2022. I'm not sure about the differences between the top 5 values that made them not sort together. I'll rerun this with `LC_ALL=C` and see what happens. That being said, it provides the info we need.

```
1892 "nginx"
1891 "nginx"
1849 "nginx"
 747 "nginx"
 147 "nginx"
   1 "nginx/1.18.0"
   1 "nginx/1.18.0 (Ubuntu)"
   1 "nginx/1.16.1"
   1 "NADA"
```

Reply:
Note that we agreed to supply this information via `service.component.*` in a separate discussion.